source/Coop/Mod/Persistence/EntityManager.cs

```diff
@@ -28,6 +28,8 @@ public class EntityManager
         private readonly RailServerRoom m_Room;
         private readonly RailServer m_Server;
         private RailServerPeer m_Arbiter;
+        public WorldEntityServer WorldEntityServer { get; private set; }
+        public bool DisableWarn { get; set; } = false;
```

Comment:
Why did you introduce that? I don't think it's a good idea to suppress warning in code. You could instead use a log receive that can filter incoming messages by logger name. Log4View for example can do that.

Reply:
@araex It is very inconvenient to catch your messages on the screen in dozens of messages about an inconsistent state.
Is there any other way not to display this on the screen?

Reply:
Why are there so many inconsistent state messages? This shouldn't happen that often, i would've guessed it happens only when new parties are spawned since that is not yet synchronized.

Reply:
I don't know, but it's always shown in 1~2 warnings per second.

Reply:
During playback, recording or at all times? This shouldn't happen. We should open an issue in that case.

Reply:
At all times.

Reply:
Ok, i opened issue #38 .